### ISSUE TITLE
Google blocks Stackedit 

### ISSUE THREAD

**Issue by tomas-nikl** [benweet/stackedit#1860]
![image](https://github.com/user-attachments/assets/2c86da33-d549-4698-ac11-6d4a173c5098)

Google blocks Stackedit because it tries to find out sensitive account information. Please correct this error.

**Comment by kierandg** [comment]
I encountered the same error 'This app is blocked This app tried to access sensitive info in your Google Account. To keep your account safe, Google blocked this access.'

**Comment by AndroidMaster25** [comment]
Same issue

**Comment by vkSegfault** [comment]
same for me

**Comment by nmcr01** [comment]
Yup, same problem.  Stackedit now effectively unusable.  This requires urgent attention or I'll be looking for a new editor

**Comment by JairoMartinezA** [comment]
> Yup, same problem. Stackedit now effectively unusable. This requires urgent attention or I'll be looking for a new editor

In my case, I am now using Obsidian. I have Google Drive in a PC folder, and from there, I created an Obsidian Vault to visualize all Markdown files.

**Comment by bugs84** [comment]
:(
Yes 
"This app is blocked. This app tried to access sensitive info in your Google Account."

Unfortunately, it is a severe issue.

**Comment by mcw256** [comment]
Anybody knows any good alternative to this app?

**Comment by AndroidMaster25** [comment]
any good alternative that support publishing to Blogger? the editor in Blogger sucks, that's why I was using Stackedit

**Comment by zxcpoiu** [comment]
https://github.com/benweet/stackedit/issues/1824#issuecomment-2633452181

**Comment by JOduMonT** [comment]
This app is blocked
This app tried to access sensitive info in your Google Account. To keep your account safe,
Google blocked this access.

<img width="835" height="307" alt="Image" src="https://github.com/user-attachments/assets/bc0108d0-4237-4445-8a4d-cae50a175dcd" />

**Comment by JayGoldberg** [comment]
To fix this issue I believe StackEdit developers need to follow this:
https://developers.google.com/identity/protocols/oauth2/production-readiness/restricted-scope-verification

I assume we can fork and run StackEdit to grant permissions on our own Google Cloud project as a "test" application that does not require review.

---
You can get around this by allowing checking the "Restrict access" when connecintg StackEdit to Google Drive. The unfortunate side effect of this is that you cannot access the files created by StackEdit outside of StackEdit; they are not visible in Drive to you or other apps.

You can see the StackEdit "Restricted" connection at https://myaccount.google.com/connections

Unfortunately, you don't even have the ability to delete data that StackEdit generates in this space, even if you "disconnect" the app. It's an invisible, unmanageable space in you GDrive.

You could write a generic POST connector for StackEdit which could call a Apps Script web endpoint to save/update data. Apps Script has easy access to Google Drive using the user's credentials. But this is pretty hacky.

**Comment by BenVoigt** [comment]
Just happened to me when I tried to make a workspace (selected Workspace in Google Drive) so I could share the file.

Well, now the app is blocked from my Google Drive account.  Which means that the synchronization/backup/version history to an app-private folder in Google Drive is now also no longer working!!!  Stackedit just always says "offline".
